docs/.vitepress/components/KSkeletonBox.vue

```diff
@@ -0,0 +1,104 @@
+<template>
```

Comment:
@mptap why did you need to recreate this component in the docs? That shouldn't be needed

Reply:
I think they somehow got autogenerated when I ran `yarn build` and `yarn docs:preview`, but I have backed them out since they aren't needed.